### PR TITLE
Feature/UUID generator

### PR DESCRIPTION
**Pull request by jmaeso**
## Description

Adds a new extension with a single command for generating a new UUID and copying it to the clipboard automatically.

## Type of change

- New extension

## Screencast

Screenshot 2021-11-26 at 15 56 59

Screenshot 2021-11-26 at 15 57 04

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/uuid-generator/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "name": "uuid-generator",
+  "title": "UUID Generator",
+  "description": "A quick way to generate UUIDs without opening the browser",
+  "icon": "command-icon.png",
+  "author": "jmaeso",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generate",
+      "title": "Generate UUID to clipboard",
```
Comment (fe9lix) [inline extensions/uuid-generator/package.json:12]:
```suggestion
      "title": "Generate UUID",
```

File: extensions/uuid-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "description": "Copy to clipboard a randomly generated UUID",
```
Comment (fe9lix) [inline extensions/uuid-generator/package.json:13]:
```suggestion
      "description": "Copy a generated UUID to the clipboard",
```

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @jmaeso, thanks. I've left some small inline comments re wording...

File: extensions/uuid-generator/package.json (hunk 2 above)
Comment (pernielsentikaer) [inline extensions/uuid-generator/package.json]:
```suggestion
      "description": "Copy a randomly generated UUID to clipboard",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @jmaeso 👋 

Only two small NIT before it's ready to get merged in 🔥 

- My suggestion about the description is not mandatory, there is just something with the wording we could look into before merging

- You could add "✅" in front of the message in the HUD so it's easier to spot with a dark background 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/uuid-generator/package.json (hunk 2 above)
Comment (jmaeso) [reply extensions/uuid-generator/package.json]: I think this was already addressed by @fe9lix but you commented into the original commit. Will discard. Suggestion into the right direction though.

**Comment by jmaeso** [comment]
✅  added to the hood message @pernielsentikaer ! Thank you for the suggestion

**Comment by pernielsentikaer** [comment]
> ✅ added to the hood message @pernielsentikaer ! Thank you for the suggestion

No problem at all, it's a fast way to improve the UX a lot — I always have a dark background myself and then the toast  “disappears” 😄

**Comment by jmaeso** [comment]
super interesting suggestion indeed! I'm using a 49" (32:9) display and first try I thought the message was not popping 😅

**Review by fe9lix — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
Congratulations with the extension @jmaeso 🔥

**Pull request merged**